plugins/woocommerce/src/Internal/Traits/RestApiCache.php

```diff
@@ -99,6 +108,30 @@ trait RestApiCache {
 	 */
 	private static string $cache_group = 'woocommerce_rest_api_cache';
 
+	/**
+	 * Cache TTL in seconds.
+	 *
+	 * @var int
+	 */
+	private static int $cache_ttl = HOUR_IN_SECONDS;
```

Comment:
Sorry for adding the same comment again 😆  I think this is not used, it was probably reintroduced by mistake.